docs/source/sections/gcp-info/GeneFlow-RNAseq.rst

```diff
@@ -0,0 +1,225 @@
+========================
+Running GeneFlow RNA-seq
+========================
+
+This GeneFlow `RNA-seq <https://www.technologynetworks.com/genomics/articles/rna-seq-basics-applications-and-protocol-299461#:~:text=RNA%2Dseq%20(RNA%2Dsequencing,patterns%20encoded%20within%20our%20RNA.>`_ workflow maps read-pairs to a reference genome and produces a transcript. 
+
+`GeneFlow <https://github.com/CDCgov/geneflow2>`_ enables modular and reproducible scientific workflows by leveraging re-usable, containerized steps. Custom workflow steps can be implemented using either Docker or Singularity containers. Additional documentation for GeneFlow can be found `here <https://geneflow.gitlab.io>`_. 

```

Comment:
Change 're-usable' to 'reusable'. Probably could be written either way, but it is 'reusable' later, so it is good to be consistent.

Reply:
Fixed